setup/ci/travis.md

```diff
@@ -5,4 +5,39 @@ sidebar:
   nav: setup
 ---
 
-Documentation for this feature is coming soon! 
+{% include toc %}
+
+You can configure Spinnaker to use [Travis
+CI](https://travis-ci.org/){:target="\_blank"} as your Continuous Integration
```

Comment:
This doesn't render properly in github view file, not sure if that is relevant for when its deployed.
![image](https://user-images.githubusercontent.com/842541/49504658-6c365a00-f87a-11e8-9520-14b7f2af634b.png)